tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -487,158 +177,17 @@ def _run(self):
-                    )
-                    out, err, rc = cmd.run()
-                    if not err:
-                        if "Connection" in out:
-                            out = (re.split(r'\s{5,}', out)[0])
-                            out = out.split(":")[1].replace(" ", "").split("/")
-                            connection_active = int(out[0])
-                            connection_count = int(out[1])
-                    NS.gluster.objects.GlobalDetails(
-                        status=out_dict['status'],
-                        volume_up_degraded=volume_up_degraded,
-                        connection_active=connection_active,
-                        connection_count=connection_count
-                    ).save()
-
-                else:
-                    # If gstatus does not return any status details in absence
-                    # of volumes, default set the status and utilization
-                    # details A sample output from gstatus if no volumes
-                    # in cluster is as below
-                    #
-                    # >gstatus -o json
-                    # This cluster doesn't have any volumes/daemons running.
-                    # The output below shows the current nodes attached to
-                    # this host.
-                    #
-                    # UUID				   Hostname  State
-                    # 6a48d2f7-3859-4542-86e0-0a8146588f31 {FQDN-1}  Connected
-                    # 7380e303-83b6-4728-918f-e99029bc1bce {FQDN-2}  Connected
-                    # 751ecb42-da85-4c3d-834d-9824d1ce7fd3 {FQDN-3}  Connected
-                    # 388b708c-a86c-4a16-9a6f-f0d53ea79a51 localhost Connected
-
-                    out_lines = stdout.split('\n')
-                    connected = True
-                    for index in range(4, len(out_lines) - 2):
-                        node_status_det = out_lines[index].split('\t')
-                        if len(node_status_det) > 2:
-                            if node_status_det[2].strip() != 'Connected':
-                                connected = connected and False
-                    if connected:
-                        NS.gluster.objects.GlobalDetails(
-                            status='healthy'
-                        ).save()
-                    else:
-                        NS.gluster.objects.GlobalDetails(
-                            status='unhealthy'
+                        NS.gluster.objects.VolumeOptions(
+                            vol_id=vol_id,
+                            options=dict1
                         ).save()
-                    NS.gluster.objects.Utilization(
-                        raw_capacity=0,
-                        usable_capacity=0,
-                        used_capacity=0,
-                        pcnt_used=0
-                    ).save()
+
+                # Sync cluster global details
+                volumes = NS.gluster.objects.Volume().load_all()
```

Comment:
Yes will add the check and because of this this PR would depend on https://github.com/Tendrl/commons/pull/692